requirements.txt

```diff
@@ -1,16 +1,12 @@
 configobj>=5.0.5
-cherrypy==8.9.1
+cherrypy>=3.6.0,<9.0
 ws4py>=0.3.2
-SQLAlchemy>=1.0.0
+SQLAlchemy>=1.1.0
 six>=1.5.2
 Jinja2>=2.7
 rpctools>=0.3.1
 logging_unterpolation>=0.2.0
 requests>=2.2.1
-pytest>=3.0.1
-mock>=1.0.1,<1.1
-Sphinx>=1.2.1
-coverage>=3.6
```

Comment:
At a glance, it looks like this has caused Travis to start failing, since we run our unit tests with the command

```
coverage run --source sideboard -m py.test sideboard
```

and we're no longer installing ``coverage`` by default.  I'm sure this is just a matter of adding another line to the ``.travis.yml`` so I'll try to get that working.